src/core/annotation.js

```diff
@@ -1535,20 +1596,42 @@ class WidgetAnnotation extends Annotation {
     );
   }
 
+  _getMKDict(rotation) {
+    const mk = new Dict(null); // `null` can probably be used here, since none of the data below will be a /Ref
```

Comment:
Please remove the comment, since I only included it to explain my reasoning as part of the review comment.